map_model/src/objects/turn.rs

```diff
@@ -110,7 +110,14 @@ impl Turn {
     // TODO What should this be for zero-length turns? Probably src's pt1 to dst's pt2 or
```

Comment:
Could you add a docstring here explaining this is a relative angle? (It really should've had something before, but I've only recently been better about adding docs...)